### DISCUSSION TITLE
Is there a way to get info on total diapers changed since start?

### DISCUSSION THREAD

**Discussion by kmanan** [babybuddy/babybuddy#617] [Q&A] [answered]
Hi, Using Baby Buddy via Home Assistant. The reports feature is really not helpful. I was wondering if there's a way to get info on total diapers changed? Essentially a total # of entries in the diapers DB?

**Comment by cdubz** [comment]
Here are some resources that may be helpful for customized reporting:

- [Baby Buddy's API documentation](http://docs.baby-buddy.net/api/) (and [openapi-schema.yml](https://github.com/babybuddy/babybuddy/blob/master/openapi-schema.yml))
- babybuddy/discussions/607
- [HA Baby Buddy Integration](https://github.com/jcgoette/baby_buddy_homeassistant) (I think? This may just be for control stuff...)

**Comment by kmanan** [comment]
Thanks for those. Not really get me what I was looking for. Not a developer either.

**Comment by tango2590** [comment, marked as answer]
You can get this data by using the API. It should give you a total count. If you wanted to narrow down between dates, you just set the `date_min` and `date_max` and the count will adjust.
<img width="641" alt="Screenshot 2023-06-23 at 16 59 14" src="https://github.com/babybuddy/babybuddy/assets/78666662/118f1752-05c3-4793-84a8-e1db954fd83f">
